ci-operator/config/openshift/origin/openshift-origin-release-4.10__okd.yaml

```diff
@@ -7,12 +7,12 @@ build_root:
 releases:
   initial:
     integration:
-      name: "4.10"
+      name: "4.12"
       namespace: origin
   latest:
     integration:
       include_built_images: true
-      name: "4.10"
+      name: "4.12"
```

Comment:
why is this changing to 4.12 in a file that contains 4.10 in the name?

Reply:
Oops, must have copypasted it wrong, sorry